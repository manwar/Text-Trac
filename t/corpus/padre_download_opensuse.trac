= openSUSE Installation Instructions =

== Installation using the Package Manager ==

== Installation from CPAN ==
 1. [http://opensuse-community.org/Repositories/11.1 Add the packman repository.]
 2. Install the package »wxWidgets-devel« and the schema »C/C++ Development«.
[[Image(yast.png)]]
 3. [wiki:DownloadCPAN Install Padre from CPAN.]

Configure [http://search.cpan.org/perldoc?CPAN#Config_Variables make_install_make_command and mbuild_install_build_command] to enable {{{sudo}}} support in CPAN.

== Still Not Working? ==
Following the instructions on this page did not lead to a successful installation?

[http://padre.perlide.org/contact.html Contact us].

We will do our best to help you.
By letting us know that something does not work you will help us, too.
When we know about it, we can improve this documentation and Padre itself.
