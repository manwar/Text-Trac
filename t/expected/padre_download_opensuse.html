<h1 id="openSUSEInstallationInstructions">openSUSE Installation Instructions</h1>
<h2 id="InstallationusingthePackageManager">Installation using the Package Manager</h2>
<h2 id="InstallationfromCPAN">Installation from CPAN</h2>
<ol><li><a class="ext-link" href="http://opensuse-community.org/Repositories/11.1"><span class="icon"></span>Add the packman repository.</a>
</li><li>Install the package »wxWidgets-devel« and the schema »C/C++ Development«.</li></ol>
<p>
[[Image(yast.png)]]
 3. <a class="wiki" href="/wiki/DownloadCPAN">Install Padre from CPAN.</a>
</p>
<p>
Configure <a class="ext-link" href="http://search.cpan.org/perldoc?CPAN#Config_Variables"><span class="icon"></span>make_install_make_command and mbuild_install_build_command</a> to enable <tt>sudo</tt> support in CPAN.
</p>
<h2 id="StillNotWorking?">Still Not Working?</h2>
<p>
Following the instructions on this page did not lead to a successful installation?
</p>
<p>
<a class="ext-link" href="http://padre.perlide.org/contact.html"><span class="icon"></span>Contact us</a>.
</p>
<p>
We will do our best to help you.
By letting us know that something does not work you will help us, too.
When we know about it, we can improve this documentation and Padre itself.
</p>